src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -1050,6 +1050,21 @@ public static SyntaxTree ParseWithRoundTripCheck(string text, CSharpParseOptions
             string sourceFileName = "",
             bool skipUsesIsNullable = false) => CreateCompilationCore(source, TargetFrameworkUtil.GetReferences(TargetFramework.Mscorlib45, references), options, parseOptions, assemblyName, sourceFileName, skipUsesIsNullable, experimentalFeature: feature);
 
+        internal static CSharpCompilation CreateNumericIntPtrCompilation(
```

Comment:
>CreateNumericIntPtrCompilation [](http://example.com/codeflow?start=42&length=30)

Consider moving to NumericIntPtrTests.cs.

Reply:
`CreateCompilationCore` is private, that's why I added this helper method here.